README.md

```diff
@@ -1,24 +1,164 @@
-# README
+# 📗 Table of Contents
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-Things you may want to cover:
 
-* Ruby version
+# 📖 cash-tracker <a name="about-project"></a>
 
-* System dependencies
+**cash-tracker** is a rails application which is designed to allow users to add and keep track of monetary transactions, creating categories to classify them and then adding them to one or moreo f them according to what they consider appropriate.
 
-* Configuration
+## 🛠 Built With <a name="built-with"></a>
 
-* Database creation
+### Tech Stack <a name="tech-stack"></a>
 
-* Database initialization
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* How to run the test suite
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Deployment instructions
+<!-- Features -->
 
-* ...
+### Key Features <a name="key-features"></a>
+
+- **Categories view:** allows the user to see the categories of transaction and how much they've spent in them, and add new ones.
+- **Payments view:** allows user to see all payments they've made within a category, and add new ones.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Deployed on Render](https://cash-tracker.onrender.com/)
+
```

Comment:
- [ ] I can't test the deployed version of your app properly, I can't sign up because the confirmation email doesn't show up in the deployed version and throws the following error in the console. Please fix that.

![image](https://user-images.githubusercontent.com/68796232/219521165-1b7e08a1-d0e6-4ec5-9ad7-3f58db2e9a46.png)

Hint: [Letteropener](https://github.com/ryanb/letter_opener) is a great gem for opening letters locally but I think it's not working in the online version.